tests/test_grad_accum.py

```diff
@@ -0,0 +1,126 @@
+import torch
+from torch import optim
+import copy
+
+from open_lm.train import train_one_epoch
+from open_lm.main import random_seed
+from open_lm.model import create_model
+from open_lm.data import get_data
+from open_lm.scheduler import cosine_lr
+from tests.testing_utils import download
+from tests.shared import MockArgs
+
+
+class TestsForAccumulatingGradients:
```

Comment:
does this run the tests you do `PYTHONPATH=. pytest tests/` from the root?

Reply:
I am, but it is running really slow... and not completing even though the functions are working. Can you take a look when you get a chance?